frontend/src/lib/components/displays/CheckpointDisplay.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+	import { GripVertical, Lock } from 'lucide-svelte';
+
+    function deleteCheckpoint() {
+        //todo
```

Comment:
Why is this not done? a generic component has no reason to be blocked, implement it